aws/resource_aws_pinpoint_app_test.go

```diff
@@ -2,6 +2,7 @@ package aws
 
 import (
 	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
```

Comment:
Nit: Go imports should be formatted as stdlib imports, new line, third party imports with each section alphabetically sorted. Tooling like `goimports` can help do this automatically in editors. 😄

Reply:
Didn't know that, Move to correct position based on other files 👍